charts/linkerd2-service-mirror/templates/service-mirror.yaml

```diff
@@ -0,0 +1,71 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: {{.Values.namespace}}
+---
```

Comment:
Please note that when using the default namespace `linkerd` and installing in a cluster with linkerd already installed, this will remove the original annotations and labels from the namespace.

Reply:
Yes I came across that today. So should I simply omit the namespace or make the template def the same as in namespace.yaml from the linkerd chart?

Reply:
We can't guarantee the service mirror service and linkerd will be installed in the same namespace (or that linkerd is present at all) so better to leave that yaml out. That'd also imply removing the `namespace` directive from the other resources, and leave the user specify that in its `kubectl` invocation.

Reply:
@alpeb Thanks I think this is the most sensible way to go. I will just leave it out